modulefiles/wcoss_dell_p3/fv3

```diff
@@ -18,8 +18,9 @@ module-whatis "loads NEMS FV3 prerequisites on Venus and Mars"
 module load ips/18.0.1.163
 module load impi/18.0.1
 module load lsf/10.1
-module load cmake/3.10.0
-module load lsf/10.1
```

Comment:
I thought we need this when submitting bsub job on dell? I have to load this module when submitting job on phase2, otherwise I got error on blaunch.

Reply:
This change updates cmake from 3.10.0 to 3.16.2, loads python/2.17.14 and removes duplicate loading of lsf/10.1